### PR TITLE
exa: init at git 2016-02-15

### PR DESCRIPTION
**Pull request by ehegnes**
This is my first work with the Nix package manager (with which I am not very familiar, since I don't use NixOS). Please let me know if the placement of my package in the Nix tree is proper or not.

I'm using a git revision due to some changes since the latest tagged release of [`exa`](https://github.com/ogham/exa) that make compilation possible without a beta or nightly Rust.

I have also added myself as a maintainer, as I'd like to continue to maintain future releases of this package.

The `exa` issue regarding the addition of this package may be seen at https://github.com/ogham/exa/issues/98.

### CODE REVIEW

**Comment by ehegnes** [comment]
Would it be more appropriate to place this package in `pkgs/tools/text/exa`?

File: pkgs/top-level/rust-packages.nix [hunk 1]
```diff
@@ -8,14 +8,14 @@
 
 let
   version = "2016-02-02";
```
Comment (wizeman) [inline pkgs/top-level/rust-packages.nix]: Please also update the date, so that we can tell how long ago was the last update. Thanks!

**Comment by wizeman** [comment]
@ehegnes Personally, I would put it in `pkgs/tools/misc/`, because that's where `coreutils` (where the real `ls` is) and `findutils` (similar type of programs) are. But I can see how the appropriate place can be subjective!

Apart from the comment about updating the date and the question about where to place it, it looks good to me!

**Comment by ehegnes** [comment]
I have made the recommended changes. Thanks for the help! :)

**Comment by wizeman** [comment]
Merged. Thanks!

**Pull request merged**